### PR TITLE
Updated team page design

### PR DESCRIPTION
**Pull request by karlgoldstraw**
## Description

Removed peoples names from the image alt. NVDA is announcing: 
“Graphic Pete Kowalczyk Pete Kowalczyk, Content designer”
“Graphic Sara Wilcox Sara Wilcox, Content designer”

## Checklist

- [ ] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ ] CHANGELOG entry
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry
- [ ] Page updated date

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -216,3 +216,26 @@ h1 + .nhsuk-lede-text {
   padding-bottom: 0;
   padding-top: 0;
 }
+
+// Team pictures used for lists
+
+.team-pictures-list{
```
Comment (chrimesdev) [inline app/styles/main.scss:222]: Please could this class be renamed `.app-team-pictures-list` and a space added before the `{`.

```
.app-team-pictures-list {
...
}

File: app/styles/main.scss [hunk 2]
```diff
@@ -216,3 +216,26 @@ h1 + .nhsuk-lede-text {
...
+    display: inline-block;
+    border: 2px solid transparent;
+    line-height: 1;
+    overflow: hidden;
+    vertical-align: middle;
+    width: 40px;
+    margin-right: 10px;
+  }
+
+}
```
Comment (chrimesdev) [inline app/styles/main.scss:241]: This file needs a new line at the end of file

File: app/styles/main.scss [hunk 3]
```diff
@@ -216,3 +216,26 @@ h1 + .nhsuk-lede-text {
...
+// Team pictures used for lists
+
+.team-pictures-list{
+  padding-left: 0;
+
+  li {
+    list-style: none;
+  }
+
+  img {
```
Comment (chrimesdev) [inline app/styles/main.scss:229]: Could the CSS for this image be tidied up to meet our CSS linting standards?

For example properties being in alphabetical order. 

```
  222:19  error  Whitespace required before {                 space-before-brace
  230:5   error  Expected `border`, found `height`            property-sort-order
  233:5   error  Expected `height`, found `border`            property-sort-order
  235:5   error  Expected `margin-right`, found `overflow`    property-sort-order
  236:5   error  Expected `overflow`, found `vertical-align`  property-sort-order
  237:5   error  Expected `vertical-align`, found `width`     property-sort-order
  238:5   error  Expected `width`, found `margin-right`       property-sort-order
  241:1   error  Files must end with a new line               final-newline
```

File: app/styles/main.scss [hunk 4]
```diff
@@ -216,3 +216,26 @@ h1 + .nhsuk-lede-text {
...
+    height: 40px;
+    border-radius: 30px;
+    display: inline-block;
+    border: 2px solid transparent;
+    line-height: 1;
+    overflow: hidden;
+    vertical-align: middle;
+    width: 40px;
+    margin-right: 10px;
```
Comment (chrimesdev) [inline app/styles/main.scss:238]: Is it possible to stick to the spacing scale (https://service-manual.nhs.uk/design-system/styles/spacing) and use spacing mixins where possible?

**Review by sarawilcox — APPROVED** [review]
Makes sense, thanks.

File: app/styles/main.scss (hunk 3 above)
Comment (henocookie) [reply app/styles/main.scss:229]: These standards: https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md#scss ? I've updated the CSS to have properties in alphabetical order but if there's anything I've missed, might be worth sharing as I've never seen these CSS standards until now

File: app/styles/main.scss (hunk 4 above)
Comment (henocookie) [reply app/styles/main.scss:238]: @chrimesdev unfamiliar with using the spacing scale for custom elements. Is this CSS okay?

```
// Team pictures used for lists

.app-team-pictures-list {
  img {
    border: 2px solid transparent;
    border-radius: 30px;
    display: inline-block;
    height: 40px;
    @include nhsuk-responsive-margin(3, "right");
    line-height: 1;
    overflow: hidden;
    vertical-align: middle;
    width: 40px;
  }
  
  @include nhsuk-responsive-padding(0, "left");
  
  li {
    list-style: none;
  }

}

```

**Review by chrimesdev — APPROVED** [review]

**Pull request merged**